config/configstore/service.go

```diff
@@ -135,8 +149,8 @@ func generateAccountKeys(keystore string, acc *Account, DID *identity.DID) (*Acc
 	return acc, nil
 }
 
-func createKeyPath(keyStorepath string, DID *identity.DID, keyName string) (string, error) {
-	tdir := fmt.Sprintf("%s/%s", keyStorepath, DID.String())
+func createKeyPath(keyStorepath string, addr *common.Address, keyName string) (string, error) {
```

Comment:
should `identity.DID` instead of `common.Address`

Reply:
The order was changed. The keys are created before DID, so DID cannot be sent via parameter

Reply:
okay 👍
Why send as a pointer ?